client/components/sidebar/sidebar-order.service.js

```diff
@@ -36,6 +37,12 @@ class SidebarOrderService {
             }
             this.SidebarMenu.addActionsMenuOption(orderItem);
         });
+        this.SidebarMenu.addActionsMenuItemClickHandler((id) => {
```

Comment:
I don't like passing argument by string id, using an Object give a lot more flexibility for future development. I would try using `.addActionsMenuItemClickHandler(({id}) => {`, but I guess it means updating the SidebarMenu.

Reply:
Yes it could be a good practice. I hesitated but for one argument... I didn't do it.
See https://twitter.com/samantha_ming/status/1002987854815846400